sdk/containerregistry/azure-containerregistry/azure/containerregistry/_authentication_policy.py

```diff
@@ -22,11 +22,13 @@ class ContainerRegistryChallengePolicy(HTTPPolicy):
     """Authentication policy for ACR which accepts a challenge"""
 
     def __init__(self, credential, endpoint, **kwargs):
-        # type: (TokenCredential, str, **Any) -> None
+        # type: (Optional[TokenCredential], str, **Any) -> None
         super(ContainerRegistryChallengePolicy, self).__init__()
         self._credential = credential
         if self._credential is None:
-            self._exchange_client = AnonymousACRExchangeClient(endpoint, **kwargs)
+            # pylint: disable=line-too-long
```

Comment:
You should just be able to add this to the one line that's too long.

Reply:
Updated